lib/cli/haste-map-react-native-packager-1-ce06e11dd572473aa7654c0c19f367c5

```diff
@@ -0,0 +1 @@
/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/delay.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/mergeAllWith.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/juxt.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/take.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/defaultsDeep.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/flattenDeep.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/isFinite.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/rangeStep.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/toUpper.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/_baseConvert.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/curryRightN.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/union.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/eachRight.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/range.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/extendAllWith.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/dropRight.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/gte.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/includes.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/conforms.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/findKey.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/pluck.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/reverse.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/tap.js":["",1483223561000,0,[]],"/home/kliment/Projects/storybook/lib/cli/test/run/react_native/node_modules/lodash/fp/sortedLastIndexBy.js":["",1483223561000,0,[]],"/home/kliment/Projects
```

Comment:
Commited accidentally?

Reply:
Committed after problems witch Continuous Integration. In commit https://github.com/storybooks/storybook/pull/3573/commits/009f6cd8d8cee7216cbf8cb4a8035d11e9b47d45

`lib/cli/haste-map-react-native-packager-1-ce06e11dd572473aa7654c0c19f367c5` updated after run command `yarn test --cli --update`

Reply:
I believe this file wasn't generated by the command

Reply:
how could i create this big file manually?
It`s generated with command.

Reply:
Sure, but probably by some other